store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStore.java

```diff
@@ -135,6 +135,7 @@ public class FederatedStore extends Store {
     private List<String> storeConfiguredGraphIds;
     @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, property = "class")
     private Map<String, BiFunction> storeConfiguredMergeFunctions;
+    private Set<Class<? extends Operation>> externallySupportedOperations = new HashSet<>();
```

Comment:
You can delete this ;) Its not used anywhere.
You set and get it, but it has not effect on `graph.getSupported operations()` 

you check for the diff before and after removing a graph. you update `supportedOperations` and you also update this superfluous object which isn't used anywhere.

Reply:
See my other comment below, this is required